README.md

```diff
@@ -1,30 +1,104 @@
+[![Go Report Card](https://goreportcard.com/badge/softonic/node-policy-webhook)](https://goreportcard.com/report/softonic/node-policy-webhook)
+[![Releases](https://img.shields.io/github/release-pre/softonic/node-policy-webhook.svg?sort=semver)](https://github.com/softonic/node-policy-webhook/releases)
+[![LICENSE](https://img.shields.io/github/license/softonic/node-policy-webhook.svg)](https://github.com/softonic/node-policy-webhook/blob/master/LICENSE)
+[![DockerHub](https://img.shields.io/docker/pulls/softonic/node-policy-webhook.svg)](https://hub.docker.com/r/softonic/node-policy-webhook)
+
+
 # node-policy-webhook
 K8s webhook handling profiles for tolerations, nodeSelector and nodeAffinity
 
+# Quick Start
+
+
+Steps to got webhook working
+
+- Deploy the webhook ( kubectl or helm )
+- Deploy NodePolicyProfile object
+- Deploy with the annotation ( below you can find it )  and watch if the pod is scheduled where was meant to
+
+## Deployment
 
-## DEVEL ENVIRONMENT
 
-### Requirements
+### Install k8s platform
 
-Install kind
+In this example we will use kind, but you can use whatever k8s platform you are comfortable with
```

Comment:
I think we can move this step somewhere else out of the quick start. In the quick start we can assume we already have a k8s cluster. What do you think?

Reply:
yes!